test/jdk/javax/swing/JButton/bug4234034.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2000, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
should be 1999 and a "," after 2023 else validate-source probably will fail, you can check mach5 -j tier1